project.py

```diff
@@ -114,6 +134,79 @@ def create_trackbars(self):
         cv.createTrackbar('lo', 'result', self.lo, 255, self.trackbar_callback)
         cv.createTrackbar('hi', 'result', self.hi, 255, self.trackbar_callback)
 
+    def check_change(self, image, mask):
+        imageOld = cv.imread("slide"+repr(self.numberOfSlides)+".png",
+                                cv.CV_LOAD_IMAGE_COLOR)
+        maskOld = cv.imread("mask.png", cv.CV_LOAD_IMAGE_GRAYSCALE)
+        if(maskOld is None):
```

Comment:
@Foxiy this is just a matter of personal preference but I would strongly prefer not to have the brackets (`(` and `)`) in here. It makes the code much more readable.